build/run.js

```diff
@@ -206,9 +206,9 @@ commands.start.rust = async function(argv) {
 commands.start.js = async function (argv) {
     await installJsDeps()
     console.log(`Building JS target.` + argv)
-    const args = targetArgs.concat([
-        `--backend-path ${paths.get_project_manager_path(paths.dist.bin)}`,
-    ])
+    // The backend path must be prepended, not appended. The `targetArgs` might include `-- …` and
```

Comment:
I dont understand this comment "The backend path must be prepended, not appended.". This sounds like it was appended before, but now it should not?

EDIT - ok, I see that this first sentence is explained by the second. Im missing a reasoning flow here. Like "The backend path is being prepended here, as appending it could break, because ..."